### PR TITLE
Update the HoC "How-To" page

### PR DESCRIPTION
**Pull request by islemaster**
[PLC-579](https://codedotorg.atlassian.net/browse/PLC-579): Creates a new [Hour of Code 2019 "How-to" page](https://hourofcode.com/how-to) at https://hourofcode.com/how-to/2019 according to [this design](https://docs.google.com/document/d/13Mj0Dd4nJB-POfz_ZXPgA1cQITfqJSdtcT_HzeNNqj8/edit?ts=5daf528b#).

### (Edit) A new approach

This change was originally rejected from Tuesday's triage meeting because the broad changes to https://hourofcode.com/how-to would break existing translations for that page, which gets a fairly large amount of non-English usage.  So instead, we are creating the updates as a new page https://hourofcode.com/how-to/2019 that we will direct US teachers to via a targeted email, and which will replace the contents of https://hourofcode.com/how-to sometime after Hour of Code, when we have time to translate it.

New testing: I've verified locally that the two pages have different content, and that https://hourofcode.com/how-to is still translated into different languages.

## Original PR Description:

[PLC-579](https://codedotorg.atlassian.net/browse/PLC-579): Updates the [Hour of Code "How-to" page](https://hourofcode.com/how-to) according to [this design](https://docs.google.com/document/d/13Mj0Dd4nJB-POfz_ZXPgA1cQITfqJSdtcT_HzeNNqj8/edit?ts=5daf528b#).

| Before | After |
| --- | --- |
| before_01 before_02 before_03 |  after_01 after_02 after_03 |

## Testing story

Content change; tested locally.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/how-to/index.md.partial [hunk 1]
```diff
@@ -6,77 +6,99 @@ nav: how_to_nav
...
+- Make sure to test tutorials on student computers or devices to ensure they work properly on browsers with sound and video. Have low bandwidth? Plan to show videos at the front of the class, so each student isn't downloading their own videos. Or try the offline tutorials.
+- Provide headphones for your class, or ask students to bring their own, if the tutorial you choose works best with sound.
+- Don't have enough devices? Use [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q). When students partner up, they help each other and rely less on the teacher. They’ll also see that computer science is social and collaborative.
 
 <img src="/images/fit-350/group_ipad.jpg">
 
-## 5. Start your Hour of Code off with an inspiring speaker or video
+### Pick a day and time
+
+People around the world join in the Hour of Code celebration during CS Education Week (December 9-13) when the latest tutorials and activities are released. But you can do an Hour of Code any day of the year!
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/how-to/index.md.partial]: Can we make these dates data-driven?  I know we have dates in pegasus/sites.v3/hourofcode.com/i18n/countries.json but not sure that we've ever used them on code.org.

**Review by breville — APPROVED** [review]

**Comment by islemaster** [comment]
I've updated the approach to this task after it was rejected from triage today.  Please take another look.

**Review by agealy — APPROVED** [review]

**Review by breville — APPROVED** [review]
Did we lose commit history for the older approach?

**Comment by islemaster** [comment]
Yes; I generated the new approach by copying the modified file and then dropping the original commit.  It looked like this (some details elided):

```sh
cp hourofcode.com/public/how-to.md.partial hourofcode.com/public/how-to/2019.md.partial
git add hourofcode.com/public/how-to/2019.md.partial
git commit -m "Create /how-to/2019"
git fetch && git rebase -i origin/staging
# Drop the original commit modifying how-to.md.partial during rebase
git push --force
```

But of course, this is `git` and nothing is ever lost. You can review the original commit here: https://github.com/code-dot-org/code-dot-org/commit/eb1faeef3e0

**Pull request merged**